static/showcases/blocked-segment.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8"/>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+  <title>Pillarbox Demo - Detect blocked segment</title>
+  <link rel="icon" type="image/x-icon" href="../../img/favicon.ico">
+  <link rel="stylesheet" href="./static-showcase.scss"/>
+  <link rel="stylesheet" href="./blocked-segment.scss"/>
+</head>
+
+<body>
+<core-demo-header></core-demo-header>
+<div class="showcase-content">
+  <h2>Detect blocked segment</h2>
+  <div class="video-container">
+    <video-js id="video-element-id" class="pillarbox-js" controls crossorigin="anonymous"></video-js>
+  </div>
+
+  <button class="showcase-btn" id="close-btn">Close this window</button>
+</div>
+
+<script type="module" data-implementation>
+  // Import the pillarbox library
+  import pillarbox from '@srgssr/pillarbox-web';
+
+  // Register a plugin to display notifications on blocked segments
+  pillarbox.registerPlugin('blockedSegmentNotification', function(options) {
+    // Get a reference to the player instance
+    const player = this;
+
+    // Wait for the player to be ready
+    player.ready(() => {
+      // Create a component for displaying the blocked segment notification
+      const blockSegmentNotification = player.addChild('component', {
+        className: 'pbw-blocked-segment-notification'
+      });
+
+      // Hide the notification initially
+      blockSegmentNotification.hide();
+
+      // Variable to store the timeout ID for clearing later
+      let timeoutId = undefined;
+
+      // Listen for the 'loadeddata' event on the player, this ensures that text tracks are available
+      player.on('loadeddata', () => {
+        // Get the text track with the ID 'srgssr-blocked-segments'
+        const blockedSegments = player.textTracks().getTrackById('srgssr-blocked-segments');
+
+        // Add a listener for the 'cuechange' event on the blocked segments text track
+        blockedSegments?.on('cuechange', () => {
+          // Check if there are active cues in the blocked segments text track
+          if (blockedSegments.activeCues.length > 0) {
+            // If there is an existing timeout, clear it
+            if (timeoutId) clearTimeout(timeoutId);
+
+            // Parse the JSON content of the active cue to get the blocked segment information
+            const blockSegment = JSON.parse(blockedSegments.activeCues[0].text);
+            const blockReason = blockSegment.blockReason;
+
+            // Update the displayed notification text with the appropriate message based on the block reason
+            blockSegmentNotification.el().textContent = options.i18n[blockReason] ?? options.i18n['UNKNOWN'];
```

Comment:
I've changed it. I was not convinced by the current labels for this kind of feature since they are always talking about `This content` but the content is being played, only a section has been skipped. Regardless it reduces the code and makes it more readable so I'm all for that.

Nevertheless we should probably showcase how to add custom localisation.